### PR TITLE
Update python tests suite

### PR DESCRIPTION
**Pull request by kdheepak**

### CODE REVIEW

**Comment by kdheepak** [comment]
Some functions in the SWIG Python interface need to be updated for use in Python. I've marked these tests to be skipped for now.

**Comment by phlptp** [comment]
Do you have a list of functions which are not working? and have you tried anything in the *.i files to get them working?

**Comment by kdheepak** [comment]
`helicsPublicationGetKey` and `helicsSubscriptionGetKey` were also having an issue. It needs to return a string instead of taking a `str` as an argument.

**Comment by phlptp** [comment]
cstring.i is already included in the helicsPython.i  but I think you saw that.  Anything specific for python should be in that file.

File: swig/helics.i [hunk 1]
```diff
@@ -17,10 +17,13 @@
 %apply helics_time_t *OUTPUT {helics_time_t*};
 %apply (char *STRING, size_t LENGTH) { (const char *data, int len) };
 %cstring_output_maxsize(char *str, int maxlen);
+%cstring_bounded_output(char* str, 1024);
```
Comment (phlptp) [inline swig/helics.i]: cstring stuff has to be in the helicsPython.i since java doesn't work with cstring.i

**Comment by kdheepak** [comment]
Merged into #233 

screen shot 2018-04-04 at 10 23 37 am

**Pull request closed without merging**